Source/WebKit/WPEPlatform/wpe/WPEDisplay.cpp

```diff
@@ -186,6 +186,7 @@ WPEDisplay* wpe_display_get_default(void)
                     return;
                 }
                 g_warning("Failed to connect to display of type %s: %s", extensionName, error->message);
+                return nullptr;
```

Comment:
```suggestion
                return;
```

Reply:
Ouch, the "return;" above is problematic, not this one -- `wpe_display_get_default` is supposed to return a WPEDisplay pointer.

The function is used from WebKitWebView:
```
WebKit/UIProcess/API/glib/WebKitWebView.cpp:        priv->display = wpe_display_get_default();
WebKit/UIProcess/API/glib/WebKitWebView.cpp:            priv->display = wpe_display_get_default();
```

Reply:
Ouch, forget my comment, misread -- we're inside a lambda.